snappy/install.go

```diff
@@ -60,6 +60,26 @@ func doUpdate(part Part, flags InstallFlags, meter progress.Meter) error {
 	return nil
 }
 
+// convertToInstalledSnaps takes a slcie of remote snaps that got
+// updated and finds the coresponding local snap parts and returns them
+func convertToInstalledSnaps(remoteUpdates []Part) ([]Part, error) {
+	installed, err := NewMetaLocalRepository().Installed()
+	if err != nil {
+		return nil, err
+	}
+
+	installedUpdates := make([]Part, 0, len(remoteUpdates))
+	for _, part := range remoteUpdates {
+		localPart := FindSnapsByNameAndVersion(part.Name(), part.Version(), installed)
+		if len(localPart) != 1 {
+			return nil, fmt.Errorf("expected one local part for the update %v got %v", part, len(localPart))
```

Comment:
I would put a comma in here.

"expected one local part for the update %v, got %v"